inc/commondbtm.class.php

```diff
@@ -3062,7 +3062,7 @@ function getLinkedItemsCount() {
    function getField($field) {
 
       if (array_key_exists($field, $this->fields)) {
-         return $this->fields[$field];
```

Comment:
This one is the most important change.
As the `getField` function is used by search (and some other places) to **display** data from db, we expect to have a string output (not a null).

I already said multiple time to not use this function when computing or testing data (like if structure) as you can have a NA (not available) string return